lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -180,7 +179,8 @@ def prepare_start
       @output_path = File.join(File.dirname(@remote_script), 'output')
       @exit_code_path = File.join(File.dirname(@remote_script), 'exit_code')
       @pid_path = File.join(File.dirname(@remote_script), 'pid')
-      @remote_script_wrapper = upload_data("echo $$ > #{@pid_path}; exec \"$@\";", File.join(File.dirname(@remote_script), 'script-wrapper'), 555)
+      @remote_script_wrapper = upload_data("echo $$ > #{@pid_path}; exec \"$@\";",
+  File.join(File.dirname(@remote_script), 'script-wrapper'), 555)
```

Comment:
This looks wrong

Reply:
@adamruzicka i can't get you point here, can you expand a little more your views?

Reply:
Well, rubocop may like it, but if suddenly indentation goes from 6 spaces to 2 in the middle of a method call, I'd say that's wrong.

I'd expect either
```ruby
      @remote_script_wrapper = upload_data("echo $$ > #{@pid_path}; exec \"$@\";",
                                           File.join(File.dirname(@remote_script), 'script-wrapper'), 555)
```

or
```ruby
  @remote_script_wrapper = upload_data(
    "echo $$ > #{@pid_path}; exec \"$@\";",
    File.join(File.dirname(@remote_script), 'script-wrapper'),
    555)
```

Having the second line indented way less than the first one, even though they form a single logical unit feels wrong.

Reply:
@adamruzicka Yes, this make sense now. Updating the changes. Thanks!